torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -893,13 +900,16 @@ RegisterOperators reg(
      // results
      DEFINE_GENERIC_OP_WITH_COMPLEX(
          aten::pow,
-         static_cast<double>(pow(a, b)),
-         static_cast<double>(pow(a, b)),
+         static_cast<double>(powWrapper(a, b)),
+         static_cast<double>(powWrapper(a, b)),
          static_cast<c10::complex<double>>(pow(a, b)),
          float,
          float,
          complex),
-     DEFINE_INT_FLOAT_OP(aten::pow, pow(a, b), float),
+     DEFINE_INT_FLOAT_OP(
+         aten::pow,
+         static_cast<double>(powWrapper(a, b)),
+         float),
      DEFINE_FLOAT_COMPLEX_OP(aten::pow, pow(a, b), complex),
      DEFINE_SCALAR_BINARY_OP_AVOID_COLLISION(
          aten::pow,
```

Comment:
Let's just wrap this case and the other ones altogether.

Reply:
I am not sure how to trigger this function call in Python and therefore, not sure how to test for this. Any thoughts?

Reply:
yeah I talked with @gmagogsfm and seems like this can be triggered when we have int ** complex, unfortunately this will crash right now because the complex case is not implemented. We can use the wrapper here and make a separate diff to implement the complex number case for scalar `binary` operations.

For the (int, int) -> int case, I don't think it's used anywhere, so we can either wrap it, or remove it.

Reply:
That makes sense, thank you!